packages/core/docs/vibe-3-changelog.md

```diff
@@ -70,6 +70,7 @@
 ### Label
 
 - `wrapperClassName` -> `className` [codemod]
+- Removed old animation
```

Comment:
I'd rephrase "old" as "slide" or "rotate", to make it more clear
Plus I think we should specify the `isAnimationDisabled` prop that was removed